packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js

```diff
@@ -98,35 +103,41 @@ export default class ScrubbingCenterController {
           this.autocomplete,
         )
         .then((data) => {
-          this.autocomplete = data;
-          this.getAllEvents('selectService');
+          this.ipsList = data.map(({ ipBlock }) => ipBlock);
+          this.selectedIp = this.ipsList;
```

Comment:
**question:** Are you trying to set the `selectedIp` with entire `ipList`?

Reply:
yes, I am